docs/framework/unmanaged-api/metadata/imetadatatables-getcolumninfo-method.md

```diff
@@ -52,6 +52,29 @@ HRESULT GetColumnInfo (
   
  `ppName`  
  [out] A pointer to a pointer to the column name.  
+ 
```

Comment:
can you also change the following:
line 3 - update ms.date value
line 37 - remove 23 of the # from Parameters so it becomes an H2 instead of H4